kubeflow-pipelines-visualization-server.advisories.yaml

```diff
@@ -162,17 +162,9 @@ advisories:
       - GHSA-5wvp-7f3h-6wmm
     events:
       - timestamp: 2024-01-19T19:23:18Z
-        type: detection
+        type: fixed
         data:
-          type: scan/v1
-          data:
-            subpackageName: kubeflow-pipelines-visualization-server
-            componentID: 1c7c197c61facfb2
-            componentName: pyarrow
-            componentVersion: 10.0.1
-            componentType: python
-            componentLocation: /usr/lib/python3.10/site-packages/pyarrow-10.0.1.dist-info/METADATA, /usr/lib/python3.10/site-packages/pyarrow-10.0.1.dist-info/RECORD, /usr/lib/python3.10/site-packages/pyarrow-10.0.1.dist-info/top_level.txt
-            scanner: grype
+          fixed-version: 2.0.5-r0
```

Comment:
> why do we think it's fixed?

The vulnerable version still exists on the package. That's because we fixed the vulnerability using a workaround (with runtime dependency injection): https://github.com/wolfi-dev/os/blob/c5360e342c9a278be1f761ce5a86ff7c49f93eb5/kubeflow-pipelines-visualization-server.yaml#L52-L53

Reply:
If it's a false positive then yes! The scanner is still finding it in this version, which is suspicious. How did we fix it? If we can provide evidence that proves the scanner is wrong, let's use `false-positive-determination` and record the evidence in the FP `type` and `note` fields.